docs/extending/skills.md

```diff
@@ -17,7 +17,7 @@ skills:
   ## Hello world (https://github.com/opsdroid/skill-hello)
   - name: hello
 ```
-You should look into the [Getting Started guide](/tutorials/introduction) to know more about skills and opsdroid configuration file.
+You should look into the [Getting Started guide](/docs/tutorials/introduction.md) to know more about skills and opsdroid configuration file.
```

Comment:
Adding /docs/ before the link will break the documentation in readthedocs, as a test try to head over to: 

[https://opsdroid.readthedocs.io/en/stable/docs/tutorials/introduction](https://opsdroid.readthedocs.io/en/stable/docs/tutorials/introduction) 

This link will fail whilst this will not

[https://opsdroid.readthedocs.io/en/stable/tutorials/introduction](https://opsdroid.readthedocs.io/en/stable/tutorials/introduction) 

Now I'm not entirely sure if we can change something on the mkdocs config file to include the `/docs` in the link so the links will work both in github and the readthedocs page. If you could have a look for me, that would be brilliant 👍